### PR TITLE
feat: [M3-6470] - Add resource links to StackScripts empty state landing state

### PR DESCRIPTION
**Pull request by cpathipa**
## Description 📝
**Add resource links to StackScripts empty state landing state**

| Before  | After   |
| ------- | ------- |
| image | image

## How to test 🧪

- Navigate to http://localhost:3000/stackscripts/account
- Validate the empty state landing page and links
- Validate Create StackScript flow

### CODE REVIEW

File: packages/manager/src/components/EmptyLandingPageResources/ResourcesLinksSubSection.tsx [hunk 1]
```diff
@@ -49,7 +49,7 @@ export const ResourcesLinksSubSection = (
 
   return (
     <StyledResourcesLinksSubSection>
-      <Typography variant="h2">
+      <Typography variant="h2" style={{ textAlign: 'left' }}>
```
Comment (abailly-akamai) [inline packages/manager/src/components/EmptyLandingPageResources/ResourcesLinksSubSection.tsx:52]: We shouldn't need this and it points to some a CSS bleed which we should address instead

Screenshot 2023-05-08 at 11 16 10 AM

File: packages/manager/src/features/StackScripts/StackScriptBase/StackScriptBase.tsx [hunk 2]
```diff
@@ -34,6 +33,7 @@ import {
 } from '../stackScriptUtils';
 import { StackScriptsRequest } from '../types';
 import withStyles, { StyleProps } from './StackScriptBase.styles';
+import { StackScriptsEmptyLandingState } from './StackScriptsEmptyLandingPage';
```
Comment (abailly-akamai) [inline packages/manager/src/features/StackScripts/StackScriptBase/StackScriptBase.tsx:36]: Let's start ordering imports where we can!

File: packages/manager/src/features/StackScripts/StackScriptBase/StackScriptsEmptyResourcesData.ts [hunk 3]
```diff
@@ -0,0 +1,72 @@
...
+
+export const youtubeLinkData: ResourcesLinkSection = {
+  links: [
+    {
+      to: 'https://www.youtube.com/watch?v=nygChMc1hX4',
+      text: 'Automate Server Deployments Using Stackscripts',
+      external: true,
+    },
+    {
+      to: 'https://jira.linode.com/browse/M3-6470#10',
```
Comment (abailly-akamai) [inline packages/manager/src/features/StackScripts/StackScriptBase/StackScriptsEmptyResourcesData.ts:52]: this links to a Jira ticket!

**Review by abailly-akamai — CHANGES_REQUESTED** [review]
Looks good! couple changes needed

File: packages/manager/src/features/StackScripts/StackScriptBase/StackScriptsEmptyResourcesData.ts (hunk 3 above)
Comment (cpathipa) [reply packages/manager/src/features/StackScripts/StackScriptBase/StackScriptsEmptyResourcesData.ts]: My bad! for some reason right click and copy link address capturing ticket address.

File: packages/manager/src/components/EmptyLandingPageResources/ResourcesLinksSubSection.tsx (hunk 1 above)
Comment (cpathipa) [reply packages/manager/src/components/EmptyLandingPageResources/ResourcesLinksSubSection.tsx]: Good call. Addressed this in the commit [611ee46](https://github.com/linode/manager/pull/9091/commits/611ee46c2c432be69bf5ec1070983a4bee2e4b1f)

File: packages/manager/src/features/StackScripts/StackScriptBase/StackScriptsEmptyResourcesData.ts [hunk 4]
```diff
@@ -0,0 +1,73 @@
...
+export const youtubeLinkData: ResourcesLinkSection = {
+  links: [
+    {
+      to: 'https://www.youtube.com/watch?v=nygChMc1hX4',
+      text: 'Automate Server Deployments Using Stackscripts',
+      external: true,
+    },
+    {
+      to:
+        'https://www.linode.com/docs/products/tools/stackscripts/guides/create/',
```
Comment (dwiley-akamai) [inline packages/manager/src/features/StackScripts/StackScriptBase/StackScriptsEmptyResourcesData.ts:53]: This is a docs guide but should be a YouTube video–can we get confirmation from UX on what link should be used here?

**Review by abailly-akamai — APPROVED** [review]

File: packages/manager/src/features/StackScripts/StackScriptBase/StackScriptsEmptyResourcesData.ts (hunk 4 above)
Comment (cpathipa) [reply packages/manager/src/features/StackScripts/StackScriptBase/StackScriptsEmptyResourcesData.ts:53]: Thank you @dwiley-akamai working with Matt regarding that. Will update the relevant link once confirmed.

**Review by jaalah-akamai — APPROVED** [review]
Approved pending updated link

**Comment by cpathipa** [comment]
> Approved pending updated link

Thank you @jaalah-akamai still waiting for the confirmation on video link.

**Comment by cpathipa** [comment]
Updated the Shell Scripts video link. Also, update title `Shell Scripts`  to `Shell Scripts Explained`" as suggested by Marketing Team.

**Pull request merged**